package.json

```diff
@@ -31,6 +31,7 @@
   "keywords": [
     "appveyor",
     "buildkite",
+    "bamboo",
```

Comment:
Sorry for nitpicking. `bamboo` should come before `buildkite`.
I try to push the fix myself but I don't have permission on your branch.
If you give me permission with I can do it myself: https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/

Reply:
@pvdlg all good! Fixed up! 🎉